gss-cb-signalling.xml

```diff
@@ -233,7 +233,7 @@ Whenever both the initiator and acceptor have provided input_channel_bindings to
           <t>
 Whenever the acceptor application has a) provided channel bindings to GSS_Accept_sec_context(), and b) not indicated support for the ret_channel_bound_flag flag, then the mechanism MUST fail to establish a security context if the initiator did not provide channel bindings data. This requirement is critical for security purposes, to make applications predating this document secure, and this requirement reflects actual implementations as deployed.</t>
           <t>
-Whenever the initiator application has a) provided channel bindings to GSS_Init_sec_context(), and b) not indicated support for the ret_channel_bound_flag flag, then the mechanism SHOULD NOT fail to establish a security context just because the acceptor failed to provide channel bindings data. This recommendation is for interoperability purposes, and reflects actual implementations that have been deployed. It is possible that not all security mechanism protocols can implement this requirement easily.</t>
+Whenever the initiator application has a) provided channel bindings to GSS_Init_sec_context(), and b) not indicated support for the ret_channel_bound_flag flag, then the mechanism MUST NOT fail to establish a security context just because the acceptor failed to provide channel bindings data. This requirement is for interoperability purposes, and reflects actual implementations that have been deployed.</t>
```

Comment:
I think the concern here was that a mechanism might do something like fold channel bindings into key derivation, so that there's really no way to make authentication work unless both sides supply the same channel bindings (including presence/absence).  So, while this change makes the third and fourth requirements consistent, it also rules out mechanisms that do that.  Maybe that's okay.

Reply:
My knowledge of what other mechanisms do and what we expect from them is not that large; I'm not strongly attached to the language either way, if you have a suggestion.